ts/windows/about/preload.ts

```diff
@@ -21,19 +21,10 @@ contextBridge.exposeInMainWorld('SignalContext', {
       environmentText.push(appInstance);
     }
 
-    let platform = '';
-    if (process.platform === 'darwin') {
-      if (process.arch === 'arm64') {
-        platform = ' (M1)';
-      } else {
-        platform = ' (Intel)';
-      }
-    }
-
     ReactDOM.render(
       React.createElement(About, {
         closeAbout: () => SignalContext.executeMenuRole('close'),
-        environment: `${environmentText.join(' - ')}${platform}`,
+        environment: environmentText.join(' - '),
```

Comment:
one small changes, can we pass in `process.platform` here so that the component depends on the data and not need to have `process` set up for it to work?

Reply:
actually I initially did exactly that, and passed in `process.arch` as well. sure thing I can revert to that approach.

Reply:
ah, looks like I don't even need to touch `About.tsx` in the first place as I could use `SignalContext.i18n` directly.

The diff should be minimal now.